frontend/src/metabase/dashboard/actions/writeback.ts

```diff
@@ -256,16 +290,12 @@ export const executeRowAction = async ({
       parameters,
     });
 
-    if (result["rows-affected"] > 0 || result["rows-updated"]?.[0] > 0) {
-      message = t`Successfully executed the action`;
-    } else if (result["created-row"]) {
-      message = t`Successfully saved`;
-    } else if (result["rows-deleted"]?.[0] > 0) {
-      message = t`Successfully deleted`;
-    } else {
-      message = t`Success! The action returned: ${JSON.stringify(result)}`;
-    }
     dispatch(reloadDashboardCards());
+    const message = getActionExecutionMessage(
+      dashcard.action as WritebackAction,
+      result,
+    );
```

Comment:
@iethree, just to be sure, is it actually possible to get here with a dashcard without an associated action?